docs/docsite/rst/getting_started_ee/introduction.rst

```diff
@@ -44,7 +44,7 @@ An Ansible user writes content for Ansible locally and wants to leverage the con
 Content separation
 ^^^^^^^^^^^^^^^^^^
 
-In situations when there is an Ansible control node or a tool such as Ansible AWX/Controller used by several users, they might want separate their content to avoid configuration and dependency conflicts.
+In situations when there is an Ansible control node or a tool such as Ansible AWX/control node used by several users, they might want separate their content to avoid configuration and dependency conflicts.
```

Comment:
```suggestion
In situations when there is an Ansible control node or a tool such as Ansible AWX/Controller used by several users, they might want separate their content to avoid configuration and dependency conflicts.
```

Reply:
This is another place where it's referring to the Ansible Controller product so this suggested edit backs out the change to control node.